stories/profile/LessonsChallenges.stories.tsx

```diff
@@ -0,0 +1,173 @@
+
+export default {
+  component: LessonsChallenges,
+  title: 'Components/LessonChallenges'
+}
+
+const completedChallenges = [
+  { challengeNumber: 1, challengeStatus: 'passed' },
+  { challengeNumber: 2, challengeStatus: 'passed' },
+  { challengeNumber: 3, challengeStatus: 'passed' },
+  { challengeNumber: 4, challengeStatus: 'passed' },
+  { challengeNumber: 5, challengeStatus: 'passed' },
+  { challengeNumber: 6, challengeStatus: 'passed' },
+  { challengeNumber: 7, challengeStatus: 'passed' },
+  { challengeNumber: 8, challengeStatus: 'passed' },
+  { challengeNumber: 9, challengeStatus: 'passed' },
+  { challengeNumber: 10, challengeStatus: 'passed' }
+]
+
+const defaultChallenges = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
+  { challengeNumber: 7, challengeStatus: 'default' },
+  { challengeNumber: 8, challengeStatus: 'default' },
+  { challengeNumber: 9, challengeStatus: 'default' },
+  { challengeNumber: 10, challengeStatus: 'default' },
+  { challengeNumber: 11, challengeStatus: 'default' },
+  { challengeNumber: 12, challengeStatus: 'default' }
+]
+
+const pendingChallenges = [
+  { challengeNumber: 1, challengeStatus: 'pending' },
+  { challengeNumber: 2, challengeStatus: 'pending' },
+  { challengeNumber: 3, challengeStatus: 'pending' },
+  { challengeNumber: 4, challengeStatus: 'pending' },
+  { challengeNumber: 5, challengeStatus: 'pending' },
+  { challengeNumber: 6, challengeStatus: 'pending' },
+  { challengeNumber: 7, challengeStatus: 'pending' },
+  { challengeNumber: 8, challengeStatus: 'pending' },
+  { challengeNumber: 9, challengeStatus: 'pending' },
+  { challengeNumber: 10, challengeStatus: 'pending' },
+  { challengeNumber: 11, challengeStatus: 'pending' }
+]
+
+const unapprovedChallenges = [
+  { challengeNumber: 1, challengeStatus: 'wrong answer' },
+  { challengeNumber: 2, challengeStatus: 'wrong answer' },
+  { challengeNumber: 3, challengeStatus: 'wrong answer' },
+  { challengeNumber: 4, challengeStatus: 'wrong answer' },
+  { challengeNumber: 5, challengeStatus: 'wrong answer' },
+  { challengeNumber: 6, challengeStatus: 'wrong answer' },
+  { challengeNumber: 7, challengeStatus: 'wrong answer' }
+]
+
+const challengesLesson6 = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
+  { challengeNumber: 7, challengeStatus: 'default' },
+  { challengeNumber: 8, challengeStatus: 'default' }
+]
+
+const challengesLesson5 = [
```

Comment:
`const challenges5=` <- this could be removed, it is extra case that is not necessary.

Reply:
> `const challenges5=` <- this could be removed, it is extra case that is not necessary.

this lesson has uniqe number of challenges, i have to hard code it. How do i go about doing that without using this? Like add an if condition on front end and look for lesson number 5 and then only show that many questions by using the same data?